doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -220,40 +222,49 @@ An B<ML-KEM-768> key can be generated like this:
 
 Equivalent calls are available for B<ML-KEM-512> and B<ML-KEM-1024>.
 
-An B<ML-KEM> private key in seed format can be converted to a key in expanded
-FIPS 203 B<dk> format by running:
+An B<ML-KEM> private key in seed format can be converted to a key in the FIPS
+203 B<dk> format by running:
 
-    $ openssl pkey -provparam ml-kem.retain_seed=no -in seed.pem -out long.pem
+    $ openssl pkey -provparam ml-kem.retain_seed=no \
```

Comment:
This works for keys, signatures would be more problematic to convert I imagine?

Reply:
Signatures are a public format, there's no issue there. Only private key formats are "controversial".